src/NPacMan.Game/GameStateMachine.cs

```diff
@@ -104,5 +107,29 @@ public GameStateMachine(IGameActions game, IGameSettings settings, GameNotificat
         public Event<GhostCollision> GhostCollision { get; private set; } = null!;
         public Event<CoinCollision> CoinCollision { get; private set; } = null!;
         public Event<PowerPillCollision> PowerPillCollision { get; private set; } = null!;
+
+        void ShowGhosts(GameState gameState)
```

Comment:
Where should this code go?

Reply:
I was thinking about a actions class but actually think it's fine in here. (Unless it gets massive)